Makefile

```diff
@@ -138,6 +139,7 @@ bin/cell:
 	      -Wl,-rpath=\$$ORIGIN \
 	      $(cell_sources) $(cell_libs)
 	cp dep/lib/$(arch)/libChakraCore.so bin
+	cp -r assets/template bin
```

Comment:
I dislike this as you generally would never install something like this into `/usr/bin` but it's tricky for in-source builds because the template is now a hard dependency for Cell.  Besides, it looks like I already do the same for `system`.  I'll allow it.

That said, shouldn't this be done *before* the `cp` you added above?

Reply:
Neither would I, but this is another case where I did the same thing as you for consistency.

Reply:
Neither would I, while it would make more sense to not copy them to bin in the minisphere and cell rules and to just copy them straight to /usr/share/minisphere in the install rule, I did that to keep consistency and because I wasn't sure if you wanted to keep it that way.

Reply:
IIRC, the reason I did this originally is so that you could just go `make` from a clean checkout and then run the engine using `bin/minisphere` without having to install it.  In hindsight I suppose I could have gotten the same result by copying the resources into `<source dir>/share`.  In any case changes to the directory structure are beyond the scope of a feature pull like this so the way you have it is fine. 👌